packages/vm/src/runBlock.ts

```diff
@@ -376,6 +385,42 @@ async function applyBlock(this: VM, block: Block, opts: RunBlockOpts) {
   return blockResults
 }
 
+export async function accumulateParentBlockHash(this: VM, block: Block) {
```

Comment:
Can this method get some code docs please?

Reply:
Do we already want to expose this method? Respectively is this generally necessary?

Maybe we keep internal until the EIP has a more stable state at least?

Reply:
👍